lib/components/home/artist_rails/__tests__/artist_card-tests.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+import 'react-native'
+import React from 'react'
+import renderer from 'react-test-renderer'
+
+import ArtistCard from '../artist_card'
+
+jest.mock('../../../spinner.js', () => 'ARSpinner')
+jest.mock('../../../opaque_image_view.js', () => 'AROpaqueImageView')
+jest.mock('../../../switch_view.js', () => 'ARSwitchView')
+
+it('renders correctly', () => {
+  const tree = renderer.create(<ArtistCard artist={artistProps().artist} />).toJSON()
+  expect(tree).toMatchSnapshot()
+})
+
+it('hides bio when metadata missing correctly', () => {
+  const props = artistProps(false)
+  const artistComponent = new ArtistCard(props)
+  const metadata = artistComponent.renderMetadata()
+
+  expect(metadata).toHaveLength(2)
+  expect(metadata[0].props.children).toEqual(props.artist.name.toUpperCase())
+  expect(metadata[1].props.children).toEqual(props.artist.formatted_artworks_count)
+})
```

Comment:
you might want to consider renaming the test to be something like `"only shows name and artwork count when bio is omitted"` as that is what the expectations are saying here 👍